config/enabled.yml

```diff
@@ -1509,3 +1509,10 @@ Rails/Validation:
 Security/JSONLoad:
   Description : 'Prefer usage of JSON.parse'
   Enabled: true
+
+Rails/HttpPositionalArguments:
```

Comment:
Move this to the proper place in the config (they should be alphabetically sorted).